client/my-sites/stats/all-time-highlights-section/index.tsx

```diff
@@ -17,94 +41,156 @@ function formatNumber( number: number | null, isCompact = false ) {
 	return Number.isFinite( number ) ? formatter.format( number as number ) : '-';
 }
 
-export default function AllTimeHighlightsSection() {
+function percentCalculator( part: number | null, whole: number | null ) {
```

Comment:
Let's reuse the percent function in the HighlightCards component for code deduplication.

Reply:
Yes! I was planning to export the percent calculator from HighlightCard directly but had no idea if it corresponds to specific component rules.

Reply:
Resolved in [ec28b43](https://github.com/Automattic/wp-calypso/pull/70001/commits/ec28b43817f4ad6dc08e52ebd0b87d08fdfdfe65).